src/drivers/power_monitor/ina226/ina226.cpp

```diff
@@ -308,6 +304,28 @@ INA226::RunImpl()
 	}
 }
 
+bool INA226::setConnected(bool state)
```

Comment:
```suggestion
bool INA226::setConnected(const bool state)
```
This emphasizes that it returns the parameter unchanged.